### ISSUE TITLE
Insufficient Color Contrast

### ISSUE THREAD

**Issue by jpdevries** [mdfw/portfolio#1]
The gray color on a white bg doesn't meet WCAG success criteria. Just needs to be a little darker

![](http://j4p.us/0P241N1s2V47/Screen%20Shot%202017-04-04%20at%209.56.20%20PM.png)

![](http://j4p.us/3o2z0k2u1Q0V/Screen%20Shot%202017-04-04%20at%209.57.31%20PM.png)
